Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "NamedDims"
 uuid = "356022a1-0364-5f58-8944-0da4b18d706f"
 authors = ["Invenia Technical Computing Corporation"]
-version = "0.2.15"
+version = "0.2.16"
```

Comment:
Seems like a feature bump

Reply:
It is a nonbreaking change.
Pre-1.0, the minor version number is only bumped for breaking changes.
(Haven't we had this discussion?)

Reply:
Looking over semver again there's nothing even stating that a major version zero patch release has to be non-breaking.

Reply:
I tend to stick to patch releases are fixes while minor releases are features

Reply:
Lets take this to slack.